services/repository.py

```diff
@@ -65,7 +97,9 @@ def get_repo_provider_service(
     data = TorngitInstanceData(
         repo=RepoInfo(
             name=repository.name,
-            using_integration=_is_repo_using_integration(repository),
+            using_integration=(
+                adapter_auth_info["selected_installation_info"] is not None
```

Comment:
This will raise a `KeyError` if `selected_installation_info` is not found. If we don't want to error, then `adapter_auth_info.get("selected_installation_info")` is safer.